### PR TITLE
Mobile menu

### PR DESCRIPTION
**Pull request by oluwajuwon8**
Add mobile menu to the hamburger on the mobile version of the portfolio.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -251,7 +254,7 @@ <h4>Skills</h4>
     </section>
     <section>
 
-    <footer class="footer">
+    <footer class="footer" id="footel">
```
Comment (bini-i) [inline index.html:257]: - [ ] Clicking the contact link from mobile side menu, does not automatically navigate to the contact section. Kindly make sure all the navigation from the side menu works.

> Hint: Please see the `id` for the footer section matches with the `id`referred from the script.

**Review by bini-i — CHANGES_REQUESTED** [review]
# Status: Changes Required :yellow_circle:            
### Hello @oluwajuwon8  :wave: 

Your project looks good and shows you have put a lot of effort into it. Kindly work on the following suggestions to further improve your project.
## Summary
- Very good job setting up linter and workflows correctly.
- Easy to read code.

## To Improve
Kindly see inline comments below :point_down: 
## Optional suggestions


_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

#### If you have any questions feel free to ask within the PR thread.
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
### HAPPY CODING :computer: :blush: 

> #### Reviewed by Binyam Hailemeskel (Code Reviewer)
> GitHub: [@bini-i](https://github.com/bini-i) 
> Twitter: [@binyamshewa](https://twitter.com/binyamshewa)
> LinkedIn: [LinkedIn](https://www.linkedin.com/in/binyam-hailemeskel-728048151/)

File: README.md [hunk 2]
```diff
@@ -18,7 +18,8 @@ To get this project in you local machine, follow the steps below:
 - Clone into your local machine
 - Run the project in you browser 
 - Inspect it to check the look in mobile format.
-Have a look at the complete live version of my [portfolio] (https://oluwajuwon8.github.io/Portfolio-Project/).
+
+  Have a look at the complete live version of my [portfolio](https://oluwajuwon8.github.io/Portfolio-Project/).
 
 ## Author
```
Comment (carlos-ssh) [inline README.md:25]: # 💾 💾 💽 [OPTIONAL]   💽 💾 💾
- Include a License to your repository.

**Review by carlos-ssh — APPROVED** [review]
# 🍾🍾 **STATUS APPROVED** 👏🏽👏🏽

Hi @oluwajuwon8 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Approved


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**